config.py

```diff
@@ -31,8 +31,8 @@ class DataSourceConfig:
     minio_path: str
     file_name: str
     files_to_download: dict[str, dict[str, str]]
-    url_minio: str | None = None
-    url_minio_metadata: str | None = None
+    url_minio: str
+    url_minio_metadata: str
```

Comment:
Why remove the default value ? I am not sure we use it for every datasource 🤔

Reply:
So far all the DAGs we refactored were using it. And since it is used by the `etl` DAG to find the relevant files I thought it was quite an important setting for the `DataSourceConfig` class.
What do you think? I can also remove the commit and wait for the class to stabilise first 🤔

Reply:
Okay good point! I think it will become complicated with RNE. Nut again we'll cross that bridge when we come to it 😅